README.rst

```diff
@@ -162,32 +162,28 @@ Transformer Engine comes preinstalled in the pyTorch container on
 From source
 ^^^^^^^^^^^
 
-First, install the prequisites.
+For JAX and Tensorflow, you need to install that dependency first:
 
 .. code-block:: bash
 
-  apt-get install ninja-build pybind11-dev
+  pip install pybind11
 
-Clone the repository and inside it type:
+Then, you can install this optional dependency:
 
 .. code-block:: bash
 
-  NVTE_FRAMEWORK=all pip install .     # Building with all frameworks.
-  NVTE_FRAMEWORK=pytorch pip install . # Building with pyTorch only.
-  NVTE_FRAMEWORK=jax pip install .     # Building with JAX only.
+  pip install ninja
 
-You can also specify which framework bindings to build. The default is pytorch only.
+Clone the repository and inside it type one of those commands:
```

Comment:
```suggestion
Install TE (optionally specifying the framework):
```

Reply:
I added the clone command line.

Reply:
Then "type one of those commands" is even more ambiguous. I agree with Kithi's suggestion.